netdata-agent-dashboard/README.md

```diff
@@ -0,0 +1,3 @@
+# Netdata Agent Dashboard
+
+This directory is used to store and document user-contributed dashboards.
```

Comment:
Since this README will serve as an index for various examples, what do you think of the following suggestion?

```suggestion
This directory is used to store and document user-contributed dashboards.

## Dashboards

- [dash-example](/netdata-agent-dashboard/dash-example/) is a single file, all-in-one page that automatically fetches graphs from all your hosts
```